.travis.yml

```diff
@@ -24,6 +24,8 @@ before_script:
 cache: pip
 
 deploy:
+    env:
```

Comment:
Does this really work?

Reply:
I hadn't opportunity to check, I relied on examples. 
I'll create a test repository and check it.

Reply:
There's nothing like this in examples. AFAIK deploy step doesn't have a separate env key.

Reply:
Yeah, it was a misunderstanding because of my lack of experience with travis.